flume-ng-sources/flume-kafka-source/src/main/java/org/apache/flume/source/kafka/KafkaSourceConstants.java

```diff
@@ -49,9 +49,13 @@
   public static final String OLD_GROUP_ID = "groupId";
 
   // flume event headers
-  public static final String TOPIC_HEADER = "topic";
+  public static final String TOPIC_HEADER_DEFAULT = "topic";
   public static final String KEY_HEADER = "key";
   public static final String TIMESTAMP_HEADER = "timestamp";
   public static final String PARTITION_HEADER = "partition";
 
+  public static final String SET_TOPIC_HEADER = "setTopicHeader";
+  public static final boolean SET_TOPIC_HEADER_DEFAULT = true;
+  public static final String TOPIC_HEADER_CONF = "topicHeader";
```

Comment:
Isn't it similar concept to the one in Sink? If not then would you mind updating the name of the DEFAULT constant to be consistent? 
(I see that KEY_HEADER looks confusing then but I guess that can be fixed in a followup commit)